changelog/seqera-enterprise/v24.2.mdx

```diff
@@ -101,6 +101,15 @@ Seqera Enterprise version 24.2 introduces new Data Studios features, global Next
 
 ## Breaking changes and warnings
 
+### Redis version change
+
+From this version of Seqera Platform: 
+
+- Redis version 6.2 or greater is required.
+- Redis version 7 is officially supported.
+
+Follow your cloud provider specifications to upgrade your instance.
```

Comment:
Given customers may not be using a managed cloud service for their Redis instance it may make sense to omit this.